.travis.yml

```diff
@@ -1,49 +1,12 @@
 language: python
-sudo: false
-cache:
-  directories:
-    - $HOME/.cache/pip
-    - $HOME/virtualenv/python$TRAVIS_PYTHON_VERSION.9
-    - $HOME/moto-s3
-python:
-  - "2.7"
-env:
-  global:
-    - CFLAGS=-O0
-    - COUCHDB_USER=""
-    - COUCHDB_PW=""
-    - TRAVIS_INSTALL="y"
-  matrix:
-    - MATRIX_TYPE=javascript BOWER=yes
-    # need node installs for app_manager.XpathValidatorTest
-    - MATRIX_TYPE=python TESTRUNNER=testrunner.GroupTestRunnerCatchall BOWER=yes NODE=yes
-    - MATRIX_TYPE=python TESTRUNNER=testrunner.GroupTestRunner0
+sudo: required
 branches:
   only:
     - master
-addons:
-  apt:
-    packages:
-    - moreutils
-    - libblas-dev
-    - liblapack-dev
 before_install:
-  - "curl http://localhost:5984/"  # print couch info
-  - "uname -a"
-  - "lsb_release -a"
-install:
-  - ".travis/misc-setup.sh"
-  - "curl -X PUT http://localhost:5984/commcarehq_test"  # this is an auth test
-  - "scripts/uninstall-requirements.sh"
-  - "time (pip install --exists-action w -r requirements/requirements.txt --timeout 60)"
-  - "pip install coveralls"
-  - "cp .travis/localsettings.py localsettings.py"
-  - ".travis/matrix-installs.sh"
-script: ".travis/matrix-runner.sh"
-after_success:
-  - coverage report
-  - coveralls
+  - "sudo docker-compose -f docker/docker-compose-test.yml build"
+  - "sudo docker-compose -f docker/docker-compose-test.yml up -d"
+  - "sudo docker-compose -f docker/docker-compose-test.yml ps"
+script: 'sudo docker-compose -f docker/docker-compose-test.yml run -e CUSTOMSETTINGS="docker.localsettings_docker_sharded" web_test python manage.py test form_processor'
```

Comment:
At first when I saw the changes to this file I thought you were changing the default hq test runner on travis over to using docker, but I see this isn't running nearly all of the tests, so I assume that's not the case.

Is it easy to tell, how does the initialization time for this docker setup compare to the initialization time for the normal hq test run on travis? That is, the time to get all the services up an running, requirements installed, etc.